config/locales/pl.yml

```diff
@@ -2,8 +2,15 @@ pl:
   auth_admin_alert: 'Strona dostępna tylko dla administratorów'
   message_of_addition: 'Record został pomyślnie dodany'
   message_of_modification: 'Record został pomyślnie zmodyfikowany'
+  message_of_deletion: 'Rekord został pomyśnie usunięty'
   choose_option: 'Wybierz z list'
 
+  maintenances:
+    index:
+      edit: 'Edycja'
```

Comment:
Popular translation which could be use in many pages add please on the top. In our case bellow 'choose_option'. It refers to 'Usun' and 'dodaj nowy wpis' too.